ksql-common/src/main/java/io/confluent/ksql/util/timestamp/TimestampExtractionPolicyFactory.java

```diff
@@ -38,11 +38,9 @@ public static TimestampExtractionPolicy create(
     final String fieldName = StringUtil.cleanQuotes(timestampColumnName.toUpperCase());
     final Field timestampField = SchemaUtil.getFieldByName(schema,
         fieldName)
-        .orElseThrow(() -> new KsqlException(String.format(
-            "No column with the provided timestamp column name in the "
-                + "WITH clause, %s, exists in the defined schema.",
-            fieldName
-        )));
+        .orElseThrow(() -> new KsqlException(
+            "The TIMESTAMP column '" + fieldName
```

Comment:
```suggestion
            "Column supplied for timestamp extractor in WITH clause does not exist in table/stream schema: " + fieldName
```
nit: grep-ability! (see, I'm learning a lot from you 😄 ) same for other exceptions that were changed

Reply:
Yep, good call.